configs/test/project.yaml

```diff
@@ -119,5 +119,8 @@ env:
 
   # Any of the above variables can be overridden in a job definition.
 
-bisect_service:
-  #pubsub_topic: /projects/project/topics/topic
+#bisect_service:
```

Comment:
Probably add comments on both these pubsub topics on the purpose of these.
Also, s/project/test-project so that create_config substitution works ? or can you create directly usable names.

e.g. projects/test-project/topics/bisect-updates ?
projects/test-project/topics/issues-updates

Reply:
Done! Will add some documentation on how to use this after I verify this works.